.github/workflows/get-merge-commit.yml

```diff
@@ -44,14 +53,62 @@ jobs:
             pull_request*)
               if commits=$(base/ci/get-merge-commit.sh ${{ github.repository }} ${{ github.event.number }}); then
                 echo -e "Checking the commits:\n$commits"
-                echo "$commits" >> "$GITHUB_OUTPUT"
+                echo "$commits" >> checked-commits
               else
                 # Skipping so that no notifications are sent
                 echo "Skipping the rest..."
               fi
               ;;
           esac
 
+      - name: Upload commits to check as an artifact
+        if: inputs.leader == true
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: checked-commits
+          path: checked-commits
+
+      - name: Get leader workflow's run ID
+        id: getLeaderRun
+        timeout-minutes: 5
```

Comment:
I think this is fine for a failure mode -- it's unlikely that we need to add anything else like a check to see if the eval workflow failed as a stopgap...? *shrug*